internal/controllers/topology/cluster/patches/variables/variables_test.go

```diff
@@ -66,12 +66,135 @@ func TestGlobal(t *testing.T) {
 						Class:   "clusterClass1",
 						Version: "v1.21.1",
 					},
+					ClusterNetwork: &clusterv1.ClusterNetwork{
+						Services: &clusterv1.NetworkRanges{
+							CIDRBlocks: []string{"10.10.10.1/24"},
+						},
+						Pods: &clusterv1.NetworkRanges{
+							CIDRBlocks: []string{"11.10.10.1/24"},
+						},
+						ServiceDomain: "cluster.local",
+					},
 				},
 			},
 			want: VariableMap{
-				"location":   toJSON("\"us-central\""),
-				"cpu":        toJSON("8"),
-				BuiltinsName: toJSON("{\"cluster\":{\"name\":\"cluster1\",\"namespace\":\"default\",\"topology\":{\"version\":\"v1.21.1\",\"class\":\"clusterClass1\"}}}"),
+				"location": toJSON("\"us-central\""),
+				"cpu":      toJSON("8"),
+				BuiltinsName: toJSONCompact(`{
```

Comment:
One additional ask, can you please indent the JSON values too? (like I did now on my PR after Fabrizio's finding)

Note: I used tabs there to make sure it's indented the same way as the regular Go code.